docs/guides/airnode/calling-an-airnode/index.md

```diff
@@ -47,19 +47,25 @@ Remix and try deploying your own Requester Contract.
 [Open in Remix<ExternalLinkImage/>](https://remix.ethereum.org/#url=https://github.com/vanshwassan/RemixContracts/blob/master/contracts/Requester.sol&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.9+commit.e5eed63a.js)
 
 ```solidity
-// SPDX-License-Identifier: MIT
+//SPDX-License-Identifier: MIT
```

Comment:
Why delete this space character? I'm not sure if there is a perfectly agreed-upon standard, but [SPDX itself has the space in there](https://spdx.dev/ids/).